Week01/Problem02/klai_02.py

```diff
@@ -0,0 +1,20 @@
+upper = 4e6
+
+
+def f(n):
```

Comment:
It's always a good idea to give functions meaningful names and documentation, so that you can easily tell what a function does either just from the name or by reading the docstring.  Take a quick look at the conventions for function docstrings [here](https://www.python.org/dev/peps/pep-0257/) and try to use this style in the future.